source/Popup.js

```diff
@@ -192,7 +182,6 @@
 		create: function () {
 			this.inherited(arguments);
 			this.animateChanged();
-			this.initialDuration = this.getComputedStyleValue('-webkit-transition-duration', '0.4s');
```

Comment:
Not that it changes the PR at all but I don't think this could have ever returned anything other the `0.4s` because `getComputedStyleValue` is guarded by `hasNode()` which would always (I think) be falsey at this point in the lifecycle.

/shrug